CMake/RealmCore.cmake

```diff
@@ -238,25 +237,41 @@ endfunction()
 
 macro(build_realm_core)
     set(core_prefix_directory "${CMAKE_CURRENT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/realm-core")
-    ExternalProject_Add(realm-core
-        PREFIX ${core_prefix_directory}
-        BUILD_IN_SOURCE 1
-        BUILD_COMMAND make -C src/realm librealm.a librealm-dbg.a ${MAKE_FLAGS}
-        INSTALL_COMMAND ""
-        CONFIGURE_COMMAND ""
-        ${USES_TERMINAL_BUILD}
-        ${ARGV}
-        )
+    if(REALM_PLATFORM STREQUAL "Android")
+      ExternalProject_Add(realm-core
+            PREFIX ${core_prefix_directory}
+            BUILD_IN_SOURCE 1
+            BUILD_COMMAND sh build.sh build-android
+            INSTALL_COMMAND ""
+            CONFIGURE_COMMAND REALM_ENABLE_ENCRYPTION=YES REALM_ENABLE_ASSERTIONS= sh build.sh config
```

Comment:
The expectation when using existing local source trees for core and sync is that the trees are already configured.